styles.css

```diff
@@ -1211,3 +1212,513 @@ li textarea {
+  color: #253858;
+  line-height: 20px;
+}
+
+.grid-item img {
+  width: 100%;
+  height: 220px;
+}
+
+.about-myself {
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  padding: 20px 20px;
+  padding-top: 100px;
+  border-top-right-radius: 25% 50px;
+}
+
+.contact-icon {
+  margin-left: 45px;
+  padding-bottom: 20px;
+}
+
+.about-myself button {
+  border-radius: 8px;
+  border: 1px solid #3d4ee4;
+  width: 170px;
+  height: 50px;
+  text-align: center;
+  padding: 10px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  background-color: white;
+  margin-bottom: 20px;
+}
+
+.flex-item6 button {
+  border-radius: 8px;
+  border: 1px solid #3d4ee4;
+  width: 140px;
+  height: 50px;
+  text-align: center;
+  padding: 10px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  background-color: white;
+  margin-top: 30px;
+}
+
+.flex-item3 img {
+  width: 100%;
+  height: auto;
+  padding-top: 5px;
+}
+
+.canopy {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.flex-item4 {
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.flex-item2 h2 {
+  color: #172b4d;
+  font-weight: bold;
+  font-size: 32px;
+}
+
+.inner-list {
+  display: flex;
+  flex-direction: column;
+}
+
+.contact {
+  background-color: #6070ff;
+  background-image: url(../images/contact.png);
+  background-size: 100%;
+  border-top-left-radius: 100px;
+  margin-top: 100px;
+}
+
+.inner-div {
+  padding-top: 100px;
+  padding-right: 30px;
+  padding-left: 20px;
+}
+
+.inner-div h2 {
+  color: white;
+  text-align: center;
+  font-size: 40px;
+  font-weight: 400;
+  padding-bottom: 0;
+}
+
+.inner-div p {
+  left: 24px;
+  top: 4802px;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+}
+
+.form {
+  padding-right: 30px;
+  padding-left: 20px;
+}
+
+li input {
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  font-family: Poppins, sans-serif;
+  border-radius: 8px;
+  width: 100%;
+  height: 50px;
+  padding-left: 10px;
+  color: #000;
+}
+
+li textarea {
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  font-family: Poppins, sans-serif;
+  border-radius: 8px;
+  width: 100%;
+  height: 150px;
+  padding-left: 10px;
+  font-size: 15px;
+  line-height: 24px;
+  padding-top: 10px;
+}
+
+.shape {
+  padding-right: 25%;
+  padding-left: 35%;
+}
+
+button:hover {
+  background: #8f9afc;
+  color: white;
+}
+
+button:active {
+  background: #0416b6;
+  color: white;
+}
+
+#portfolioPopup {
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgb(76, 78, 82, 0.7);
+  display: none;
+}
+
+ul.works-flex-item.titles-container {
+  margin: 10px;
+}
+
+.works-flex {
+  padding: 20px;
+  background: #fff;
+  margin: 25px 20px;
+  border-radius: 10px;
+  max-width: 100%;
+}
```

Comment:
Great job @Nowembabazi 🤝

- [ ] Kindly fix the vertical scroll on your mobile popup window. This will make your design a close replica of the design provided in the Figma template.
![vertical-scroll](https://user-images.githubusercontent.com/63917725/189389688-36949f18-0f1a-4a5b-ba28-3903d5638940.PNG)